docs/navigate/devops-testing/toc.yml

```diff
@@ -503,6 +503,12 @@ items:
             href: ../../core/deploying/native-aot/interop.md
           - name: Build native libraries
             href: ../../core/deploying/native-aot/libraries.md
+          - name: iOS-like platforms
```

Comment:
I would put this and future OS-specific topics at the end of "Native AOT deployment model" list.

Reply:
Thanks, fixed all your remarks.